core/primitives/src/trie_key.rs

```diff
@@ -254,6 +269,14 @@ impl TrieKey {
                     + std::mem::size_of_val(index)
             }
             TrieKey::BandwidthSchedulerState => col::BANDWIDTH_SCHEDULER_STATE.len(),
+            TrieKey::OutgoingReceiptGroupsQueueData { .. } => {
+                col::OUTGOING_RECEIPT_GROUPS_QUEUE_DATA.len() + std::mem::size_of::<u64>()
+            }
+            TrieKey::OutgoingReceiptGroupsQueueItem { index, .. } => {
+                col::OUTGOING_RECEIPT_GROUPS_QUEUE_ITEM.len()
+                    + std::mem::size_of::<u64>()
+                    + std::mem::size_of_val(index)
+            }
```

Comment:
I like `size_of_val` more than `size_of`, it's impossible to make a mistake, it always calculates the actual size of the field. I only used `size_of` for the `shard_id` because I wasn't sure how `size_of_val` is going to behave with the `ShardId` wrapper.